### PR TITLE
Openshift pipeline

### PR DESCRIPTION
**Pull request by alvarolobato**
@reviewbybees esp @ndeloof @cyrille-leclerc

### CODE REVIEW

**Comment by cyrille-leclerc** [comment]
Sounds good, can you squash all the commits into 1 before we merge?

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+        step([$class: 'WarningsPublisher', consoleParsers: [[parserName: 'Maven']]])
+        step([$class: 'JUnitResultArchiver', testResults: '**/target/surefire-reports/TEST-*.xml'])
+        step([$class: 'JavadocArchiver', javadocDir: 'gameoflife-core/target/site/apidocs/'])
+
+        // Use fully qualified hudson.plugins.checkstyle.CheckStylePublisher if JSLint Publisher Plugin or JSHint Publisher Plugin is installed
+        step([$class: 'hudson.plugins.checkstyle.CheckStylePublisher', pattern: '**/target/checkstyle-result.xml'])
+        // In real life, PMD and Findbugs are unlikely to be used simultaneously
+        step([$class: 'PmdPublisher', pattern: '**/target/pmd.xml'])
+        step([$class: 'FindBugsPublisher', pattern: '**/findbugsXml.xml'])
+        step([$class: 'AnalysisPublisher'])
```
Comment (oleg-nenashev) [inline Jenkinsfile]: I've thought we have simpleBuildSteps for it... CC @amuniz

**Comment by alvarolobato** [comment]
Done.

File: Jenkinsfile (hunk 1 above)
Comment (cyrille-leclerc) [inline Jenkinsfile]: @oleg-nenashev we have for ArtifactArchiver, I have not found steps for others

**Pull request merged**